dbms/src/TiDB/Schema/SchemaBuilder.cpp

```diff
@@ -530,6 +530,7 @@ void SchemaBuilder<Getter, NameMapper>::applyDiff(const SchemaDiff & diff)
         break;
     }
     case SchemaActionType::SetTiFlashReplica:
+    case SchemaActionType::UpdateTiFlashReplicaStatus:
```

Comment:
I don't understand why `UpdateTiFlashReplicaStatus` will change the replica number